docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.8"
+services:
+
+  mongomock:
+    build:
+      context: .
+    image: mongomock:latest
+    working_dir: /project
+    command: tox
+    environment:
+      - TEST_MONGO_HOST=mongodb://mongo/mock
+    volumes:
+      - .:/project
+    depends_on:
+      - mongo
+
+  mongo:
+    # The image chosen here should be kept in sync with the .travis.yml mongo version
+    image: mongo:4.4.0
+    ports:
+      - "27017:27017"
```

Comment:
I don't think you need to expose the port to the host: only the mongomock container should access it.